src/index.html

```diff
@@ -12,7 +12,55 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="stars__body">
+    <div class="wrapper">
+      <div class="stars stars--0">
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
+        <div class="stars_star"></div>
```

Comment:
Each 'stars_star' div should have a meaningful class name that reflects the content rather than the style or tag name.